handlers/event.js

```diff
@@ -81,8 +82,16 @@ module.exports.get = async (event, ctx, callback) => {
 module.exports.update = async (event, ctx, callback) => {
 
   const data = JSON.parse(event.body);
-  const timestamp = new Date().getTime();
 
+  const params = {
```

Comment:
Nit: It's a little confusing that params is declared here but used later in the code after the `updateDB()` function is declared. Would prefer moving this down or moving `updateDB()` down

Reply:
Good call